docs/developer/decision-records/2023-05-25-template-repository/README.md

```diff
@@ -0,0 +1,90 @@
+# Template and `.github` Repositories
+
+## Decision
+
+Two new repositories will be created within the EDC GitHub organization:
+- `.github` repository for common and universal documents,
```

Comment:
nit: i would highlight, that the `.github` repo will contain files that are _the same_ for all components, whereas the template repos contain files that must be _adapted/modified_ by each component.